.wordlist.txt

```diff
@@ -143,6 +164,8 @@ LXDE
 LXQt
 macOS
 mangas
+memcache
+MariaDb
```

Comment:
Should be MariaDB -> https://en.wikipedia.org/wiki/MariaDB

Reply:
And not sure where `memcache` is used, but in text I would say "memory cache"/"memory caching" and "memcached" would be a specific implementation (https://www.memcached.org/), but at least in PHP we use APCu instead.

Reply:
Changed MariaDb -> MariaDB

Reply:
memchache is used there: software/webserver_stack/#lamp-webserver-apache2-mariadb-php.  
@MichaIng, @fpetru: Your turn for discussion. :-)

Reply:
Changed to memory caching in the webserver_stack.md file.